app.js

```diff
@@ -11,39 +13,98 @@ app.use(express.json());
 
 // Iteration 1 - Connect to MongoDB
 // DATABASE CONNECTION
+const MONGODB_URI = "mongodb://127.0.0.1:27017/express-mongoose-recipes-dev"
+
+mongoose.connect(MONGODB_URI)
+.then((x)=> console.log(`Connected to Mongo! Database name: "${x.connections[0].name}`))
+.catch((err) => console.log("Error connecting to Mongo", err))
 
 
 
 // ROUTES
-//  GET  / route
+//  GET  / route - This is just an example route
 app.get('/', (req, res) => {
-    res.send("<h1>Lab | Express Mongoose Recipes</h1>");
+    res.send("<h1>LAB | Express Mongoose Recipes</h1>");
 });
 
 
-//  Iteration 4 - Create recipe route
+//  Iteration 3 - Create a Recipe route
 //  POST  /recipes route
-
-
-//  Iteration 5 - Read all recipes
+app.post('/recipes', (req,res)=>{
+    console.log(req.body)
+    Recipe.create(req.body)
```

Comment:
Something that we can  do in order to keep the code easy to read is destructuring the properties from it  and other persons knowns which properties we need to create a Recipie is something like this
```suggestion
const { title, instructions, level, ingredients, image ,duration, isArchived, created} = req.body
  Recipe.create({
    title,
    instructions,
    level,
    ingredients,
    image,
    duration,
    isArchived,
    created,
  })
```